tutorials/eeg-coding-culture.rst

```diff
@@ -0,0 +1,14 @@
+Coding culture at EEG
+=====================
+
+This talk was given by `Philip Hackstock <https://iiasa.ac.at/staff/philip-hackstock>`_
+on the 17th of January 2022 for the EEG group at the TU Vienna.
```

Comment:
You should introduce the abbreviation. Also, adding a link to the EEG website would be helpful for context.
```suggestion
on the 17th of January 2022 for the *Energy Economics Group* (EEG) group at the TU Vienna.
```